ParselyTracker/Sampler.swift

```diff
@@ -64,6 +65,7 @@ class Sampler {
                   key: key,
                   accumulatedTime: TimeInterval(0),
                   totalTime: TimeInterval(0),
+                  firstSampleTime: Date(),
```

Comment:
Nitpicky, perhaps -- I wonder if `firstSampleTime` and `lastSampleTime` sound too much like endpoints when put next to each other, and something like `mostRecentSample` (or something more brief) could help highlight that we aren't trying to define some interval of time where sampling takes place.

Reply:
I can see better names for `lastSampleTime`, but changing that seems out of scope for this pull request. I've added it to https://github.com/Parsely/engineering/issues/3042